app/features/history/history_filter/tags/SelectTagsModal.js

```diff
@@ -0,0 +1,375 @@
+// TODO: consider splitting this component into two different ones rather than using if statements everywhere
```

Comment:
- [ ] why did we hvae to copy paste this one again? could we not use an existing one?

Reply:
The only other existing one IS 1rm's select tags modal, In this case the only other option would have been to literally use it. I suppose we could refactor it, but once again time was of the essence here.